Drivers/src/ADCS747.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * ADCS747.cpp
+ *
+ *  Created on: Feb 24, 2022
+ *      Author: John Carr
+ */
+
+#include <ADCS747.hpp>
+
+namespace SolarGators {
+namespace Drivers {
+
+ADCS747::ADCS747(SPI_HandleTypeDef* hspi, GPIO_TypeDef* cs_port, uint16_t cs_pin):
+    hspi_(hspi), cs_port_(cs_port), cs_pin_(cs_pin)
+{
+
+}
+
+ADCS747::~ADCS747() {
+  // TODO Auto-generated destructor stub
+}
+
+void ADCS747::Init()
+{
+  // Just need 16 clock cycles result of this is invalid
```

Comment:
Is this just clearing `ADCS747`'s SPI buffer?

Reply:
For turning on the IC we need to send it 16bits, not sure if it clears the spi buffer.